pkg/constants/constants.go

```diff
@@ -52,4 +52,7 @@ const (
 
 	// ControlInfoMsgKey - control info message key.
 	ControlInfoMsgKey = "ControlInfo"
+
+	// hubOfHubsLocalResource - local-resource label
```

Comment:
annotation, not label

Reply:
I have updated to use `label`. updated anywhere except for `hub-of-hubs-gitops` repo. will do it soon.